index.html

```diff
@@ -8,55 +8,335 @@
+              </ul>
+            </div>
+          </div>
+          <div class="card-btn">
+            <a href="#" class="see-project">See Project</a>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="card-block">
+            <div class="card-title">Professional Art Printing Data</div>
+            <div class="card-post">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </div>
+            <div class="card-tags">
+              <ul>
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">ruby</a></li>
+              </ul>
+            </div>
+          </div>
+          <div class="card-btn">
+            <a href="#" class="see-project">See Project</a>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="card-block">
+            <div class="card-title">Professional Art Printing Data</div>
+            <div class="card-post">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </div>
+            <div class="card-tags">
+              <ul>
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">ruby</a></li>
+              </ul>
+            </div>
+          </div>
+          <div class="card-btn">
+            <a href="#" class="see-project">See Project</a>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="card-block">
+            <div class="card-title">Professional Art Printing Data</div>
+            <div class="card-post">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </div>
+            <div class="card-tags">
+              <ul>
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">ruby</a></li>
+              </ul>
+            </div>
+          </div>
+          <div class="card-btn">
+            <a href="#" class="see-project">See Project</a>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!--SECTION ABOUT ME-->
+    <div class="section-about-me">
+      <div class="aboutme-block">
+        <div class="about-me-heading">About me</div>
+        <div class="about-me-desc">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </div>
+        <div class="resume-btn">
+          <a href="#" id="resume">Get My Resume</a>
+        </div>
+      </div>
```

Comment:
- [ ] Please the button here should follow the button styling: Active, Hover and Pressed
![image](https://user-images.githubusercontent.com/74744787/159130876-10e427c7-4f58-475f-bd81-984beadce883.png)

Reply:
Fixed